samples/cpp/matchmethod_orb_akaze_brisk.cpp

```diff
@@ -0,0 +1,162 @@
+#include <opencv2/opencv.hpp>
+#include <vector>
+#include <iostream>
+
+using namespace std;
+using namespace cv;
+
+static void help()
+{
+    cout << "\n This program demonstrates how to detect compute and match ORB BRISK and AKAZE descriptors \n"
+        "Usage: \n"
+        "  ./matchmethod_orb_akaze_brisk <image1(../data/basketball1.png as default)> <image2(../data/basketball2.png as default)>\n"
+        "Press a key when image window is active to change algorithm or descriptor";
+}
+
+
+
+int main(int argc, char *argv[])
+{
+    vector<String> typeDesc;
+    vector<String> typeAlgoMatch;
+    vector<String> fileName;
+    help();
+    system("cd");
```

Comment:
change this line to `int ret = system("cd");` to remove the warning about the return value not being used